### DISCUSSION TITLE
Where the Media Fragments?  请教是起始结束范围吗？

### DISCUSSION THREAD

**Discussion by kuku652** [aidenlx/media-extended#380] [Q&A] [answered]
更新后，看到这个梦寐以求的功能（知识类卡片笔记神器）
我以为是在一段长视频中指定起始结束范围。

但是现在只找到指定起始范围，如何指定一段呢？

**Comment by HugleDong** [comment]
## How to set up the media fragment so that we can replay it in a range? 

We can't find this feature in the Obsidian command, nor can we find it in the official documentation

Looping the media fragment is a great function in some situation, like language learing. We yearn for it~~

Anyway, thanks for the effort, we appreciate that.

**Comment by aidenlx** [comment, marked as answer]
https://mx.aidenlx.site/docs/v4/tutorials/capture-study-note#create-a-media-clip-step
